waterbutler/providers/rushfiles/provider.py

```diff
@@ -264,9 +264,11 @@ async def _upload_request(self, stream, path, created):
             metadata = await self.metadata(path)
         request_body = json.dumps({
             'RfVirtualFile': {
+                'InternalName': path.identifier if not created else '',
                 'ShareId': self.share['id'],
                 'ParrentId': path.parent.identifier,
                 'EndOfFile': str(stream.size),
+                'Tick': 0,
```

Comment:
Although required, `Tick` is ignored/overwritten on the RF side. Settings it to 0 is fine, but please add a comment explaining why.

E.g. "Required, but ignored so can be set to any value"